addons/l10n_us_account/__manifest__.py

```diff
@@ -0,0 +1,16 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': 'United States - Accounting',
+    'website': 'https://www.odoo.com/documentation/saas-17.2/applications/finance/fiscal_localizations.html',
+    'icon': '/account/static/description/l10n.png',
+    'countries': ['us'],
+    'version': '1.0',
+    'category': 'Accounting/Localizations/Account Charts',
+    'description': """
+    """,
+    'depends': ['l10n_us', 'account'],
+    'installable': True,
+    'auto_install': ['account'],
```

Comment:
What? Please don't!
This will install directly `l10n_us` on all databases that install `account`!

Reply:
@KangOl 

Not since https://github.com/odoo/odoo/pull/144586

Reply:
We are doing this to allow install `l10n_us` automatically on US dbs.
but If a user installs the `l10n_us_account`, it will install `l10n_us`  for him as it's part of its dependencies.

Reply:
This is ... disconcerting. Really subtle.

But whatever, this IMProvement should not target the stable version.

Reply:
It's more of a FIX than an IMP. Why shouldn't it target stable ?

People starting new databases in the US and installing account won't have `l10n_us` installed by default which is a bit annoying because many other US modules depend on it (`l10n_us_check_printing`, `l10n_us_1099`, ...).

However, because of https://github.com/odoo/odoo/pull/161709#discussion_r1570910384 I don't think we should do this in the `l10n_us` module, but instead in a new module `l10n_us_account` which will be auto installed if country == US and the account module is installed. At the point, we'll see that the dependency `l10n_us` is missing and it'll be installed too (as well as the other `l10n_us_...` modules I mentioned).